bakerydemo/static/css/bootstrap.min.css

```diff
@@ -2,5 +2,5 @@
+/*# sourceMappingURL=bootstrap.min.css.map */
```

Comment:
It feels safer for us to remove this too if we remove the `.map` file?

Reply:
Already done!